src/fstab-generator/fstab-generator.c

```diff
@@ -644,6 +645,12 @@ static int add_mount(
                 }
         }
 
+        if (flags & MOUNT_QUOTA) {
+                r = generator_hook_up_quotaservice(dest, where, target_unit);
```

Comment:
just call thus `generator_hook_up_quota()`, i.e. drop the `service` in the name, please, w don't have that for the others such functions either.